packages/react-composites/tests/browser/chat/app/index.tsx

```diff
@@ -73,6 +79,16 @@ function App(): JSX.Element {
     }
   }, [adapter]);
 
+  const fileDownloadHandler: FileDownloadHandler = (userId, fileData): Promise<URL | FileDownloadError> => {
+    return new Promise((resolve) => {
+      if (fileData.name !== 'Unauthorized.pdf') {
```

Comment:
Keying off of filename here feels unclean. Can you add a new field `fileData.shouldFailDownload` or something that makes this more robust?

Reply:
Oh wait, `fileData` comes from the sent message, so it's part of the exported API (can't add fields to that that).
In that case, I'd recommend adding a new `qArg` to the test sample app that says "fail all file downloads". And then load `page1` with that `qArg`.